exercises/isogram/.meta/generator/isogram_case.rb

```diff
@@ -6,9 +6,19 @@ def workload
     indent_lines(
       [
         "string = #{input.inspect}",
-        "#{assert} Isogram.is_isogram?(string)"
+        "#{assert} Isogram.isogram?(string), #{failure_message}"
       ], 4
     )
   end
 
+  private
+
+  def failure_message
+    %Q("Expected '#{expected}', '#{input}' #{type}")
+  end
+
+  def type
+    "#{expected ? 'is' : 'is not'} an isogram"
```

Comment:
For me it is a little bit easier to read it as `Q("Expected '#{expected}', #{reason}")` rather than `%Q("Expected '#{expected}', '#{input}' #{reason}")` if say I was just skimming through it.